assayist/client/query.py

```diff
@@ -0,0 +1,46 @@
+# SPDX-License-Identifier: GPL-3.0+
+
+import neomodel
+
+from assayist.common.models.content import Build
+from assayist.client.error import NotFound
+
+
+def set_connection(neo4j_url):
+    """
+    Set the Neo4j connection string.
+
+    :param str neo4j_url: the Neo4j connection string to configure neomodel with
+    """
+    neomodel.db.set_connection(neo4j_url)
+
+
+def get_container_content_sources(build_id):
+    """
+    Get the sources used by the content in the container image.
+
+    :param int build_id: the Koji build's ID
+    :return: a list of source URLs
+    :rtype: list
+    """
+    build = Build.nodes.get_or_none(id_=str(build_id))
+    if not build:
+        raise NotFound('The requested build does not exist in the database')
```

Comment:
Just curious, why not use `get()` and get the exception raising for free?

Reply:
This is a client library, so I'm trying to abstract the fact that we use neomodel by raising custom exceptions that are available in the library. If I didn't, the user of this library would have to have knowledge of how neomodel works and would do the following to catch the exception:

```python
from assayist.client import query
from assayist.common.models.content import Build
try:
    query.get_container_content_sources(1234123)
except Build.DoesNotExist as e:
    # Do something here
    pass
```

It's not terrible, so we could use that approach and just document that's how you catch the exception, but the message is not very helpful:
```
neomodel.core.BuildDoesNotExist: (BuildDoesNotExist(...), "{'id_': '1234123'}")
```